go/kbtest/insecure/insecure.go

```diff
@@ -0,0 +1,16 @@
+package insecure
```

Comment:
you should add 

    // +build !production

to this file

Reply:
Do I also need to add a version of this file that will build on production and export same function but empty?